ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/SyncTargetManager.java

```diff
@@ -121,7 +124,8 @@ private CompletableFuture<Optional<EthPeer>> confirmPivotBlockHeader(final EthPe
     task.assignPeer(bestPeer);
     return ethContext
         .getScheduler()
-        .timeout(task)
+        .timeout(task, Duration.ofSeconds(MAX_QUERY_RETRIES_PER_PEER * 5 + 1))
```

Comment:
Is the 5 that is referred to here coming from the timeout in `AbstractRetryingPeerTask`? It would be nicer to create a constant there and use it here instead of having a magic value.